### PR TITLE
Enemy fixes

### PR DESCRIPTION
**Pull request by dariormt**

### CODE REVIEW

File: Assets/Enemy/EnemyBehaviourData.cs [hunk 1]
```diff
@@ -1,5 +1,14 @@
-using System.Collections.Generic;
+using System;using System.Collections.Generic;
```
Comment (ynks) [inline Assets/Enemy/EnemyBehaviourData.cs:1]: this should be two lines

File: Assets/Player/PlayerWeaponManager.cs [hunk 2]
```diff
@@ -26,15 +26,25 @@ public class PlayerWeaponManager : MonoBehaviour
     
     private bool _isWeaponHeld;
     private bool _wantsToThrowOrGet;
+    [SerializeField] ProgressBar _progressBar;
```
Comment (ynks) [inline Assets/Player/PlayerWeaponManager.cs:29]: pls dont put underscore on serializefield variables and use explicit private whenever you can

File: Assets/Enemy/EnemyBehaviourData.cs (hunk 1 above)
Comment (dariormt) [reply Assets/Enemy/EnemyBehaviourData.cs:1]: tf happened there??????????

**Review by ynks — CHANGES_REQUESTED** [review]

**Review by ynks — APPROVED** [review]
image

**Pull request merged**